docs/src/components/pricing/PricingTable.tsx

```diff
@@ -647,15 +588,31 @@ const rowHeaders: Record<string, React.ReactNode> = {
   // charts - features
   'charts/legend': <ColumnHead label="Legend" href="/x/react-charts/legend/" />,
   'charts/tooltip': <ColumnHead label="Tooltip" href="/x/react-charts/tooltip/" />,
-  'charts/mouse-zoom': <ColumnHead label="Zoom on mouse" />,
+  'charts/zoom-and-pan': <ColumnHead label="Zoom and Pan" href="/x/react-charts/zoom-and-pan/" />,
   'charts/export': <ColumnHead label="Export" />,
   // charts - datagrid
   'charts/cell-with-charts': (
     <ColumnHead label="Cell with chart" href="/x/react-data-grid/custom-columns/#sparkline" />
   ),
   'charts/filter-interaction': <ColumnHead label="Row filtering" />,
   'charts/selection-interaction': <ColumnHead label="Range selection" />,
-  'tree-view/tree-view': <ColumnHead label="Tree View" href="/x/react-tree-view/" />,
+  // Treeview - components
+  'tree-view/simple-tree-view': (
+    <ColumnHead label="Simple Tree View" href="/x/react-tree-view/simple-tree-view/items/" />
+  ),
+  'tree-view/rich-tree-view': (
+    <ColumnHead label="Rich Tree View" href="/x/react-tree-view/rich-tree-view/items/" />
+  ),
+
+  // Treeview - advanced features
+  'tree-view/inline-editing': (
+    <ColumnHead label="Inline editing" href="/x/react-tree-view/rich-tree-view/editing/" />
+  ),
+  'tree-view/drag-to-reorder': (
+    <ColumnHead label="Drag to reorder" href="/x/react-tree-view/rich-tree-view/ordering/" />
```

Comment:
Maybe simply

```suggestion
    <ColumnHead label="Ordering" href="/x/react-tree-view/rich-tree-view/ordering/" />
```

https://github.com/mui/material-ui/pull/42328#discussion_r1612486486

Reply:
Must we match the page title, though?
Because I think that just "Ordering" without more context doesn't communicate well what the feature is about, so we're basically forcing users to leave the pricing page, whereas "drag to reorder" feels something that you can understand a bit better from the label itself.

Reply:
If it's true, with the pricing page, is it different with

<img width="281" alt="SCR-20240902-topi" src="https://github.com/user-attachments/assets/4cd4d718-2820-4a8d-8b7b-8905e42303c7">

or?

<img width="644" alt="SCR-20240902-touz" src="https://github.com/user-attachments/assets/ab842953-6912-4d2c-8e4c-a9624e59e18b">